lib/cocoapods/installer/analyzer.rb

```diff
@@ -185,6 +185,10 @@ def update_mode
         end
       end
 
+      def podfile_dependencies
```

Comment:
is this public? does it it need to be?

Reply:
nope, it's private and can stay that way